oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/FileIOUtils.java

```diff
@@ -395,7 +394,7 @@ public int compare(String s1, String s2) {
      * @deprecated use {@link org.apache.jackrabbit.oak.commons.io.FileLineDifferenceIterator} instead
      */
     @Deprecated(since = "1.20.0", forRemoval = true)
-    public static class FileLineDifferenceIterator extends AbstractIterator<String> implements Closeable {
+    public static class FileLineDifferenceIterator extends com.google.common.collect.AbstractIterator<String> implements Closeable {
```

Comment:
```suggestion
    public static class FileLineDifferenceIterator extends AbstractIterator<String> implements Closeable {
```

Reply:
Yes, that's a case where we need to preserve API compat. This will be removed at a later point.